ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/Search.tsx

```diff
@@ -142,102 +171,124 @@ function Search(props): any {
             alert('Please delete special characters in the conditions!');
             return;
         }
-        // according [input val] to change searchFilter list
-        const allFilterConditions = searchInputVal.trim().split(';');
         const newSearchFilter: any = [];
+        const str = searchInputVal.trim();
+        // user input: string and this string don't include [: < > ≠]
+        if (str.includes('>') || str.includes('<') || str.includes(':') || str.includes('≠')) {
```

Comment:
We are currently using "≠" character for non-equal filter?

Reply:
by origin design, we use `≠` to show these status `!=`. but seems `!=` is better. and I seems `!==` is more smooth

Reply:
![image](https://user-images.githubusercontent.com/61399850/167096389-8eed4213-5d27-4c4f-bcc3-a0dc73909ec8.png)